woocommerce/wc-admin
# Fix use of feature checks and remove deprecated method calls

Fixes #6685 

Removes deprecated calls to `Loader::is_feature_enabled()` and also updates this to check if a feature is enabled as opposed to existing.

### Detailed test instructions:

1. On a new site with the nav disabled run the `wc_admin_daily` cron event.
2. Check that the nav nudge note is added, but the feedback note has not been added.
3. Enable the navigation.
4. Run the `wc_admin_daily` cron event again.
5. Check that the nav feedback note is added.
6. Generally smoke test other features to make sure they are not disabled.
